components/global/SnowCmip6.vue

```diff
@@ -0,0 +1,349 @@
+    },
+    {
+      color: '#b2e2e2ff',
+      label:
+        '&ge; 20 kg m&#8315;&sup2; s&#8315;&sup1;, &lt; 40 kg m&#8315;&sup2; s&#8315;&sup1;',
+    },
+    {
+      color: '#66c2a4ff',
+      label:
+        '&ge; 40 kg m&#8315;&sup2; s&#8315;&sup1;, &lt; 60 kg m&#8315;&sup2; s&#8315;&sup1;',
+    },
+    {
+      color: '#2ca25fff',
+      label:
+        '&ge; 60 kg m&#8315;&sup2; s&#8315;&sup1;, &lt; 80 kg m&#8315;&sup2; s&#8315;&sup1;',
+    },
+    { color: '#006d2cff', label: '&ge; 80 kg m&#8315;&sup2; s&#8315;&sup1;' },
+  ],
+  swe: [
+    { color: '#edf8fbff', label: '&ge;0 mm, &lt;75 mm' },
+    { color: '#b2e2e2ff', label: '&ge;75 mm, &lt;150 mm' },
+    { color: '#66c2a4ff', label: '&ge;150 mm, &lt;225 mm' },
+    { color: '#2ca25fff', label: '&ge;225 mm, &lt;300 mm' },
+    { color: '#006d2cff', label: '&ge;300 mm' },
+  ],
+}
+
+const mapId = 'sea_ice_cmip6'
+mapStore.setLegendItems(mapId, legend)
+</script>
+
+<template>
+  <section class="section xray">
+    <div class="content is-size-5">
+      <XrayIntroblurb resolution="100" unit="km" cmip="6" beta />
+      <h3 class="title is-3">Snow, CMIP6</h3>
+      <p class="mb-6">
+        The map below shows modeled snow variables for the month of March using
+        the MIROC6 model at 25-year intervals from 1950&ndash;2000. Intervals
+        from 2025&ndash;2100 are based on the SSP5-8.5 emissions scenario.
+      </p>
+
+      <MapBlock :mapId="mapId" crs="EPSG:3572" class="mb-6">
+        <template v-slot:layers>
+          <h4 class="title is-4 mb-3">March Precipitation as Snow</h4>
+          <MapLayer
+            v-for="layer in prsn_layers"
+            :mapId="mapId"
+            :layer="layer"
+            :key="layer.id"
+            :default="layer.default"
+          >
+            <template v-slot:title>{{ layer.title }}</template>
+          </MapLayer>
+          <h4 class="title is-4 mb-3">March Snow Water Equivalent</h4>
+          <MapLayer
+            v-for="layer in swe_layers"
+            :mapId="mapId"
+            :layer="layer"
+            :key="layer.id"
+            :default="layer.default"
+          >
+            <template v-slot:title>{{ layer.title }}</template>
+          </MapLayer>
+        </template>
+      </MapBlock>
+
+      <p>
+        Enter a location below to see a chart of the monthly snow water
+        equivalent and precipitation as snow for a point location using the
+        selected model, emissions scenario, and month. After entering a
+        location, links will be provided where you can download the data that is
+        used to populate the chart.
+      </p>
+
+      <Gimme :bbox="[-180, 45, 180, 90]" />
+      <Cmip6MonthlyChartControls
+        defaultModel="MIROC6"
+        defaultMonth="12"
+        :datasetKeys="['prsn', 'swe']"
+      />
+      <Cmip6MonthlyChart
+        label="Precipitation as Snow in kg m⁻² s⁻¹"
+        units="kg m⁻² s⁻¹"
+        dataKey="prsn"
+      />
```

Comment:
```suggestion
        dataKey="prsn"
        class="mb-5"
      />
```
This just adds a little bit of spacing between the charts like other CMIP6 data x-ray items with multiple charts.